src/index.html

```diff
@@ -1,18 +1,71 @@
 <!doctype html>
+
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
```

Comment:
The attributes of the meta tag are correctly formatted. Each attribute starts on a new line with a 2-space indentation related to the tag. However, the closing bracket of the tag is not on the same level as the opening one. Please fix this.